src/routes/Chat.tsx

```diff
@@ -92,6 +103,10 @@ function SingleMessage(props: {
                         return {
                             type: "invoice",
                             status: "paid",
+                            original_message: props.dm.message.replace(
```

Comment:
can you call this `message_without_invoice` or something. `original_message` feels like the whole message and will be confusing when we haven't looked at this code in 6 months

Reply:
good call